test/integration/cli/quiet.test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env node, mocha */
+/* eslint-disable func-names, prefer-arrow-callback, no-loop-func, max-len */
+
+import { assert } from 'chai';
+import path from 'path';
+import { exec } from 'child_process';
+
+const fixtureDir = path.relative(process.cwd(), path.join(__dirname, 'fixture'));
+const binPath = path.relative(process.cwd(), path.join('bin', '_mocha'));
+const test = path.join(fixtureDir, 'simple/simple.js');
+
+describe('cli --quiet', function () {
+  it('does not show info messages', function (done) {
+    exec(`node ${binPath} --quiet "${test}"`, (err, stdout) => {
+      assert.isNull(err);
+      assert.notInclude(stdout, 'WEBPACK');
```

Comment:
Maybe add an `assert.include(stdout, 'passing');` or something like this to ensure output is not completely empty and contains the expected output?

Reply:
✅